plugins/newsdownloader.koplugin/feeds.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> JSON is a pain imo. When manually editing config files I like to:

Yes, that's why I didn't mention it when I was talking about which DSL to pick for KOReader :) I am using json mostly as an example for how far simple data structure can get you.

JSON is also not designed for better readability and the encode/decoding performance is really bad. I would not pick JOSN for any of my project unless I need to integrate with a web UI or any legacy REST system.

I also wrote a lot of XML 6 years ago when I was writing chapters using docbook. Oh god, it was painful.